docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md

```diff
@@ -0,0 +1,308 @@
+* [otelcol.auth.basic][]
+* [otelcol.exporter.loki][]
+* [otelcol.exporter.otlp][]
+* [otelcol.exporter.prometheus][]
+* [otelcol.processor.batch][]
+* [otelcol.receiver.otlp][]
+* [prometheus.remote_write][]
+
+[loki.write]: {{< relref "../reference/components/loki.write.md" >}}
+[otelcol.auth.basic]: {{< relref "../reference/components/otelcol.auth.basic.md" >}}
+[otelcol.exporter.loki]: {{< relref "../reference/components/otelcol.exporter.loki.md" >}}
+[otelcol.exporter.otlp]: {{< relref "../reference/components/otelcol.exporter.otlp.md" >}}
+[otelcol.exporter.prometheus]: {{< relref "../reference/components/otelcol.exporter.prometheus.md" >}}
+[otelcol.processor.batch]: {{< relref "../reference/components/otelcol.processor.batch.md" >}}
+[otelcol.receiver.otlp]: {{< relref "../reference/components/otelcol.receiver.otlp.md" >}}
+[prometheus.remote_write]: {{< relref "../reference/components/prometheus.remote_write.md" >}}
+
+## Before you begin
+
+* Ensure that you have basic familiarity with instrumenting applications with
+  OpenTelemetry.
+* Have a set of OpenTelemetry applications ready to push telemetry data to
+  Grafana Agent Flow.
+* Identify where Grafana Agent Flow will write received telemetry data.
+* Be familiar with the concept of [Components][] in Grafana Agent Flow.
+
+[Components]: {{< relref "../concepts/components.md" >}}
+
+## The Pipeline
+
+We start with the config from the previous guide:
+
+```river
+otelcol.receiver.otlp "example" {
+  grpc {
+    endpoint = "127.0.0.1:4317"
+  }
+
+  http {
+    endpoint = "127.0.0.1:4318"
+  }
+
+  output {
+    metrics = [otelcol.processor.batch.example.input]
+    logs    = [otelcol.processor.batch.example.input]
+    traces  = [otelcol.processor.batch.example.input]
+  }
+}
+
+otelcol.processor.batch "example" {
+  output {
+    metrics = [otelcol.exporter.otlp.default.input]
+    logs    = [otelcol.exporter.otlp.default.input]
+    traces  = [otelcol.exporter.otlp.default.input]
+  }
+}
+
+otelcol.exporter.otlp "default" {
+  client {
+    endpoint = "my-otlp-grpc-server:4317"
+  }
+}
+```
+
+This represents the pipeline:
+```
+Metrics, Logs, Traces: OTLP Receiver → batch processor → OTLP Exporter
+```
+
+We will be implementing the following pipeline in this guide:
+
+```
+Metrics: OTel → batch processor → prometheus remote write
+Logs: OTel → batch processor → Loki exporter
+Traces: OTel → batch processor → OTel exporter
+```
+
+## Grafana Tempo
+
+[Grafana Tempo][] is an open-source, easy-to-use, scalable distributed tracing backend. Tempo can ingest OTLP directly, and we can use the OTLP exporter to send the traces to Tempo.
+
+```river
+otelcol.exporter.otlp "default" {
+  client {
+    endpoint = "tempo-server:4317"
+  }
+}
+```
+
+To use Tempo with basic-auth, which is the case with GrafanaCloud Tempo, you must use the [otelcol.auth.basic][] component. For example, you can get the Tempo config from the “details” page of Tempo in the [GrafanaCloud Portal][]:
```

Comment:
```suggestion
To use Tempo with basic-auth, which is the case with GrafanaCloud Tempo, you must use the [otelcol.auth.basic][] component. You can get the Tempo config from the Tempo **Details** page in the [GrafanaCloud Portal][]:
```

Do I read this right that you MUST use basic-auth for GrafanaCloud Tempo? OR do you have other auth options as well?